### PR TITLE
Script to download tinycore linux.

### PR DESCRIPTION
**Pull request by pnasrat**
Work towards automated testing #24.

Download tinycorelinux from a mirror and patch.

Update hyperkitrun.sh to call script and also explicitly call make if
needed.

Tested commands by using hyperkitrun.sh

Signed-off-by: Pris Nasrat <pnasrat@gmail.com>

### CODE REVIEW

**Comment by ijc25** [comment]
LGTM. Thanks!

File: hyperkitrun.sh [hunk 1]
```diff
@@ -20,7 +22,20 @@ LPC_DEV="-l com1,stdio"
...
+  fi
+fi
+
 # Linux
+if [ ! -f "$KERNEL" ]; then
+ pushd test
+ ./tinycore.sh
+ popd
+fi
 build/com.docker.hyperkit $ACPI $MEM $SMP $PCI_DEV $LPC_DEV $NET $IMG_CD $IMG_HDD $UUID -f kexec,$KERNEL,$INITRD,"$CMDLINE"
```
Comment (ijc25) [inline hyperkitrun.sh]: Would be nice to use the new `$HYPERKIT` here, but I'm not going to let that stop me pushing the merge button.

File: test/tinycore.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+# These are binaries from a mirror of
+#  http://tinycorelinux.net
+# with the following patch applied:
+# Upstream source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+#BASE_URL="http://www.tinycorelinux.net/"
+
+BASE_URL="http://distro.ibiblio.org/tinycorelinux/"
```
Comment (ijc25) [inline test/tinycore.sh]: Is this temporary or do we expect `ibiblio.org` to be more reliably available than `tinycorelinux.net`?

**Comment by ijc25** [comment]
I had a couple of late-breaking very minor comments above, but nothing to prevent a merge, thanks!

**Comment by pnasrat** [comment]
Thanks! I'll do a follow up when I add a simple automated test which
addresses the comments.

**Pull request merged**